packages/sdks/overrides/react-native/src/functions/transform-style-property.ts

```diff
@@ -166,13 +166,28 @@ function omit<T extends object>(obj: T, ...values: (keyof T)[]): Partial<T> {
 const inRange = (value: number, min: number, max: number) =>
   value >= min && value <= max;
 
+function extractVarDefaultValue(value: string): string {
+  // Regular expression to find var() expressions
+  const varRegex = /var\(--[^,]+?,\s*([^)]+)\)/;
+
+  // Function to replace var() with its fallback
+  let newValue = value;
+  let match: string[] | null;
+  while ((match = newValue.match(varRegex))) {
+    newValue = newValue.replace(match[0], match[1].trim());
+  }
+
+  return newValue;
+}
```

Comment:
Can we add some tests for this regex `while` loop? 😅

I know this is in an `overrides`, so it might have to move to packages/sdks/src/functions/extract-var-default-value.ts` so that it can be tested.

Reply:
ah yeah agree on tests, and on the issues with overrides being why i didn't add it. i hate the idea of react native specific code bloating the other SDKs though, but I guess no better way. its copy pasta'd from webapp so we know is working well already, but will add tests to guard from future fixes breaking things

Reply:
yup, I know what you mean about bloat.

But since:
- the file won't be imported by anything else in other SDK
- all SDKs have a bundling step


therefore it won't end up in bundle output of any other SDK. So this is safe to do 👍🏽

Reply:
yup, updated. at first i misread your comment to move that into an existing file vs its own, its in its own with tests now